docs/resources/connector.md

```diff
@@ -47,5 +47,6 @@ resource "twingate_connector" "aws_connector" {
 Import is supported using the following syntax:
 
 ```shell
+terraform import twingate_remote_network.aws_network UmVtb3RlTmV0d29yazozOTkwNw==
```

Comment:
i don't think we need to add these imports to documentation, its kinda obvious that a network has to be present in order for the connector to work

Reply:
removed it